lang-python/cython-0.29/autobuild/defines

```diff
@@ -0,0 +1,9 @@
+PKGNAME=cython-0.29
+PKGSEC=python
+PKGDES="C extensions for Python (Version 0.29)"
+PKGDEP="python-2 python-3 setuptools"
+
+PKGPROV=cython
+PKGCONFL="cython>=3.0.0"
```

Comment:
No, it's in fact conflicting with all `cython` versions, since older `cython` versions has the same file lists as `cython-0.29`.